### PR TITLE
Jira AT-1046 and Jira AT-1151

### PR DESCRIPTION
**Pull request by PieterJP**
**SpectrometerHDF5OutputFile.h and cpp** 
(AT-1151):
1. Remove observation information from a file.
2. Add attributes for:
3. antenna name
4. antenna diameter
5. half power beamwidth
6. antenna longitude
7. antenna latitude
8. antenna altitude
9. observer
10. Conversion from DMS to degrees
**(AT-1046):**
1. Add noise-diode tables to metadata

### CODE REVIEW

File: SpectrometerHDF5OutputFile.cpp [hunk 1]
```diff
@@ -102,14 +107,11 @@ cSpectrometerHDF5OutputFile::cSpectrometerHDF5OutputFile(const std::string &strF
...
+
     //Read antenna config information from ini file
-    pt::ptree ObsInfo;
-    pt::ini_parser::read_ini("ObservatoryInfo.ini", ObsInfo);
-    setAntennaName(ObsInfo.get<string>("Antenna.name"));
-    setAntennaLatitude(ObsInfo.get<string>("Antenna.latitude"));
-    setAntennaLongitude(ObsInfo.get<string>("Antenna.longitude"));
-    setAntennaAltitude(ObsInfo.get<string>("Antenna.altitude"));
-    setAntennaDiameter(ObsInfo.get<string>("Antenna.diameter"));
     setAntennaBeamwidth(ObsInfo.get<string>("Antenna.beamwidth"));
```
Comment (rishadebrahim) [inline SpectrometerHDF5OutputFile.cpp:115]: It looks like you removed ObsInfo and parsing of the ini file but you are still using it here.

File: SpectrometerHDF5OutputFile.cpp [hunk 2]
```diff
@@ -2723,3 +2909,93 @@ void cSpectrometerHDF5OutputFile::addAttenuationADCChan1(int64_t i64Timestamp_us
...
+    vector<string> rows;
+    string line;
+    while (getline(inFile, line))
+    {
+        rows.push_back(line);
+        lineCount++;
+    }
+    string row;
+    const string seperator = ",";
+    //string item;
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp:2967]: Remove this commented code

File: SpectrometerHDF5OutputFile.cpp [hunk 3]
```diff
@@ -2723,3 +2909,93 @@ void cSpectrometerHDF5OutputFile::addAttenuationADCChan1(int64_t i64Timestamp_us
...
+    const int dimension = 2;
+    double data[lineCount][dimension];
+
+    int entries = 0;
+    //while (getline(inFile, row))
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp:2972]: Remove this commented code

File: SpectrometerHDF5OutputFile.cpp [hunk 4]
```diff
@@ -2414,46 +2518,128 @@ void cSpectrometerHDF5OutputFile::setAppliedPointingModel(const string &strModel
...
 
-void cSpectrometerHDF5OutputFile::setAntennaBeamwidth(const string &strAntennaBeamwidth_deg)
+void cSpectrometerHDF5OutputFile::setObservationInfo(const string &strObservationInformation)
 {
     boost::shared_lock<boost::shared_mutex> oLock(m_oAppendDataMutex);
-    //Only a single set of values
-    sprintf(m_oAntennaConfiguration.m_chaAntennaBeamwidth, strAntennaBeamwidth_deg.c_str());
-}
+    // Antenna\_info:\_Latitude:\_05:45:02.4696\_Longitude:\_00:18:17.9172\_Altitude:\_116\_Diameter:\_32\_Name\_of\_observer:\_ATP_Operator
+    string strAntennaName           = "TDB";  // Must still be addes in SCS
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp:2549]: Surely these TBDs shouldn't be here?

File: SpectrometerHDF5OutputFile.cpp [hunk 5]
```diff
@@ -2414,46 +2518,128 @@ void cSpectrometerHDF5OutputFile::setAppliedPointingModel(const string &strModel
...
+        {
+            dDiameter = stod(strObservationInformation.substr(start, found - start));
+        }
+    }
+    found = strObservationInformation.find(strObserverName);
+    if (string::npos != found)
+    {
+        start = found + strObserverName.length();
+        found = strObservationInformation.length();
+        //if (string::npos != 0)
```
Comment (rohanschwartz) [inline SpectrometerHDF5OutputFile.cpp:2624]: Remove this commented code

**Pull request closed without merging**